app/utils/SplashLoadingScreen.ts

```diff
@@ -1,24 +1,30 @@
-// @ts-nocheck File not migrated fully to TS
-/**
- * Created by jakubniezgoda on 06/13/2017.
- */
-
-const SPLASH_DIV = $('div.splashPage');
+const SPLASH_DIV: HTMLElement | null = document.querySelector('div.splashPage');
 const ACTIVE_CLASS = 'active';
-const FADE_OUT_SPEED = 'slow';
+const FADE_OUT_SPEED = 300;
 
 export default class SplashLoadingScreen {
+    static fadeOut(element: HTMLElement) {
```

Comment:
Can we make it a `private` method? Also, this always will refer to `SPLASH_DIV`, so we could ditch the parameter now :slightly_smiling_face: 

```suggestion
    private static fadeOut() {
```